recipes/jina-hubble-sdk/LICENSE

```diff
@@ -0,0 +1,193 @@
+Copyright 2020-2021 Jina AI Limited.  All rights reserved.
```

Comment:
I don't see this license anywhere upstream. We can't just add a new license here, it needs to come from upstream. The link to the Apache-2.0 license upstream redirects to a 404 for a non-existent license file, while the license metadata in the upstream `setup.py` says "Proprietary".

We need an accurate license file actually endorsed by upstream in order to know whether we can legally redistribute this package.

Reply:
We have add add accurate [license file](https://github.com/jina-ai/jina-hubble-sdk/blob/main/LICENSE) in our rep `jina-hubble-sdk`  but It's from version v0.24.0, is there a problem.